### PR TITLE
changed chameleon ssd optimizer download link to a stable official link

### PR DESCRIPTION
**Pull request by pan-long**
changed chameleon ssd optimizer download link to a stable official link

### CODE REVIEW

File: Casks/chameleon-ssd-optimizer.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version :latest
   sha256 :no_check
 
-  url 'http://chameleon.alessandroboschini.it/download.php'
+  url 'http://chameleon.alessandroboschini.com/cdownload.php'
   appcast 'http://chameleon.alessandroboschini.it/sparkle/profileInfo.php'
   name 'Chameleon SSD optimizer'
   homepage 'http://chameleon.alessandroboschini.it/'
```
Comment (vitorgalvao) [inline Casks/chameleon-ssd-optimizer.rb]: Same with `homepage`.

File: Casks/chameleon-ssd-optimizer.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
   version :latest
   sha256 :no_check
 
-  url 'http://chameleon.alessandroboschini.it/download.php'
+  url 'http://chameleon.alessandroboschini.com/cdownload.php'
   appcast 'http://chameleon.alessandroboschini.it/sparkle/profileInfo.php'
```
Comment (vitorgalvao) [inline Casks/chameleon-ssd-optimizer.rb]: Looks like the `appcast` also works on `.com`, now. Could you please change it?

File: Casks/chameleon-ssd-optimizer.rb (hunk 1 above)
Comment (pan-long) [inline Casks/chameleon-ssd-optimizer.rb]: yes, sure

**Comment by phinze** [comment]
LGTM, thanks!

**Pull request merged**